scripts/monitoring/ops-metric-client.py

```diff
@@ -64,6 +64,14 @@ def run(self):
 
         self.metric_sender.send_metrics()
 
+    @staticmethod
+    def adjust_type(val):
+        """ convert float values to float type. keep string as is """
+        try:
+            return float(val)
```

Comment:
My preference is for #1, but I could live with #2.  I don't really like #3.

Could we do #1, but allow for an override so that we could give it "1.3" and set the type to string?

In other words, if we specify the type, use it. If we don't specify a type, then attempt to detect it.